.github/workflows/build.yml.in

```diff
@@ -58,6 +58,17 @@ jobs:
         run: |
           lake exe lint-style
 
+      - name: "check the formatting of the PR title and for missing or inconsistent labels"
+        # TODO: skip this check if the PR is in draft stage!
+        shell: bash # just in case
+        run: |
+          set -eo pipefail
+          title="${{github.event.pull_request.title}}"
```

Comment:
I think Actionlint suggests using `env` in the workflow step, cf. [the example here](https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable)).